desktop/src/app/components/navbar/warnings/modals/convert-field-data-modal.component.ts

```diff
@@ -24,6 +24,9 @@ export class ConvertFieldDataModalComponent {
     public inputTypeLabel: string;
 
     public convertAll: boolean;
+    public countAffected: string;
```

Comment:
This should be a number (it doesn't need to be a string to be included in the HTML template)